src/www/ui/api/Controllers/LicenseController.php

```diff
@@ -463,4 +470,26 @@ public function deleteAdminLicenseCandidate($request, $response, $args)
     }
     return $response->withJson($resInfo->getArray(), $resInfo->getCode());
   }
+
+  /**
+   * Get all license standard comments
+   *
+   * @param Request $request
+   * @param ResponseHelper $response
+   * @param array $args
+   * @return ResponseHelper
+   */
+  public function getAllLicenseStandardComments($request, $response, $args)
+  {
+    if (!Auth::isAdmin()) {
+      $error = new Info(403, "You are not allowed to access the endpoint.", InfoType::ERROR);
+      return $response->withJson($error->getArray(), $error->getCode());
+    }
```

Comment:
Get for this endpoint should be allowed for all users as it is used at file clearing page as well.

Reply:
![snap](https://github.com/fossology/fossology/assets/18077542/ec105cb7-5552-412c-8f82-9015999d80f1)